server/controllers/api/assessmentDayDataController/index.js

```diff
@@ -121,6 +131,56 @@ const AssessmentDayDataController = {
         }
       }
 
+      const isAssessmentInDb = await AssessmentModel.findOne(
+        appDb,
+        assessmentQuery
+      )
+
+      if (!isAssessmentInDb) {
+        const newAssessmentProperties = AssessmentModel.withDefaults({
+          name: assessment,
+        })
+        await AssessmentModel.upsert(
+          appDb,
+          assessmentQuery,
+          newAssessmentProperties
+        )
+      }
```

Comment:
since we're upserting do we need to check if the document exists? We should be able to call upsert every time.

Reply:
[0872fb6](https://github.com/AMP-SCZ/dpdash/pull/726/commits/0872fb6ba250403a2178df163c52f903ce4d1518)